src/components/ChatRoom.tsx

```diff
@@ -0,0 +1,80 @@
+import { Show, For, createSignal, useContext, onMount } from "solid-js";
+import log from "loglevel";
+import { AppContextContext } from "../AppContextProvider";
+import { Group, Message, MessageParams } from "../types";
+import * as api from '../api'
+import "./ChatRoom.css"
+
+type ChatRoomProps = {
+  group: Group
+};
+
+
+export default function ChatRoom(props: ChatRoomProps) {
+  let divRef: HTMLDivElement | undefined
+  const [formValue, setFormValue] = createSignal('')
+  const [appState] = useContext(AppContextContext);
+  const [openChatRoom, setOpenChatRoom] = createSignal(true);
+  
+  onMount(() => { 
+    divRef?.scrollIntoView();
+  })
+
+  async function handleSendMessage() { 
+    try {
+      if (formValue())  {
+        const message = {
+          groupId: props.group?.id,
+          id: appState?.user?.id,
+          name: appState?.user?.name,
+          email: appState?.fbUser?.email,
+          photoURL: appState?.fbUser?.photoURL,
+          text: formValue(),
+        }
+        await api.chat.addMessage(message as MessageParams)
+        setFormValue('')
+      }
+    } catch (e) {
+      log.error(e);
+    } finally { 
+      divRef?.scrollIntoView({ behavior: 'smooth', block: "end"});
```

Comment:
You also don't need this with the above change

Reply:
removed and replaced this with `createEffect`